ecs-cli/modules/commands/cluster/cluster_command.go

```diff
@@ -68,7 +68,7 @@ func clusterUpFlags() []cli.Flag {
 		},
 		cli.BoolFlag{
 			Name:  command.CapabilityIAMFlag,
-			Usage: "Acknowledges that this command may create IAM resources.",
+			Usage: "Acknowledges that this command may create IAM resources. Not needed if custom instance role is specified.",
```

Comment:
Can we make this wording similar to the --capability-iam caveat?

"Acknowledges that this command may create IAM resources. Required if --instance-role is not specified."